detector-data/detectors/HPS_PhysicsRun2019-v2-FEE-Pass0/HPS_PhysicsRun2019-v2-FEE-Pass0.lcdd

```diff
@@ -0,0 +1,10070 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lcdd xmlns:lcdd="http://www.lcsim.org/schemas/lcdd/1.0" xmlns:xs="http://www.w3.org/2001/XMLSchema-instance" xs:noNamespaceSchemaLocation="http://www.lcsim.org/schemas/lcdd/1.0/lcdd.xsd">
+  <header>
+    <detector name="HPS_SurAliUCMS_ForProd_BOTAlign_iter9" />

```

Comment:
Do these detector names need to be the same as the name of the det in the directory tree? I think this would just require regenerating this lcdd with the compact from this directory?

Reply:
They don't need to, but they really should be.

Reply:
It needs to be the same since once you try to reconstruct any events simulated with this detector, it will look for the detector named in the lcdd file. It will either not be found, or the wrong detector will be picked up. And the reconstruction will most likely crash since the SimTrackerHits won't be inside the Si sensors.

Reply:
Most of the time we specify the detector anyway right? So it doesn't need to be the same.

Reply:
Anyway, I think we are all in agreement that it should be changed.

Reply:
Thank you for spotting this! 
Should be fixed.